src/themes/animation.scss

```diff
@@ -59,6 +59,16 @@
   animation-name: fade-in-down;
 }
 
+// Fade-in-down-wait
+@-webkit-keyframes fade-in-down { 0% { opacity: 0; -webkit-transform: translateY(-20px); } 100% { opacity: 1; -webkit-transform: translateY(0); } }
+@-moz-keyframes    fade-in-down { 0% { opacity: 0; -moz-transform: translateY(-20px);}     100% { opacity: 1; -moz-transform: translateY(0); } }
```

Comment:
Do we really need these vendor-prefixed properties? 
Won't webpack generate it ?